src/main/java/org/cascadebot/cascadebot/utils/votes/VoteMessageType.java

```diff
@@ -20,11 +20,6 @@
     /**
      * This vote type will use discord regional indicator letters up to the amount specified in the builder.
      */
-    LETTERS,
-
-    /**
-     * This vote type will give you complete control over what reactions you want to use .
-     */
-    CUSTOM
+    LETTERS
```

Comment:
Where's the custom type gone?

Reply:
Because the custom type is hard to do now that they're persistent the way it was being done before. At some point I might return to this.